.github/workflows/unit_tests.yaml

```diff
@@ -225,23 +225,25 @@ jobs:
       if: ${{ needs.changes.outputs.with_coverage == 'true' }}
       env:
           HOME: /home/spack-test
+          SPACK_TEST_SOLVER: original
       run: |
           whoami && echo $HOME && cd $HOME
           git clone https://github.com/spack/spack.git && cd spack
           git fetch origin ${{ github.ref }}:test-branch
           git checkout test-branch
-          share/spack/qa/run-unit-tests
+          bin/spack unit-test -x
     - name: Run unit tests (only package tests)
       if: ${{ needs.changes.outputs.with_coverage == 'false' }}
       env:
           HOME: /home/spack-test
           ONLY_PACKAGES: true
+          SPACK_TEST_SOLVER: original
       run: |
           whoami && echo $HOME && cd $HOME
           git clone https://github.com/spack/spack.git && cd spack
           git fetch origin ${{ github.ref }}:test-branch
           git checkout test-branch
-          share/spack/qa/run-unit-tests
+          bin/spack unit-test -x -m "not maybeslow" -k "package_sanity"
```

Comment:
Is there a reason we're not running the shell tests anymore?

Reply:
The libtool change was a quick workaround a month ago to not select libtool 2.4.2 in DAGs.  The real issue has been then fixed in https://github.com/spack/spack/pull/25585 but since the package update also made sense in itself I kept it. I can extract that into another PR if it makes more sense.

Reply:
> Is there a reason we're not running the shell tests anymore?

It's because of point 1. in the description (note that we are not running shell tests anymore only in the CentOS 6 tests):

> CentOS 6 still uses the original concretizer as it can't connect to the buildcache due to issues with ssl (bootstrapping from sources requires a C++14 capable compiler)

I encountered issues with CentOS 6 in CI due apparently to using an old Python interpreter. Since it is my understanding (or maybe it's just a strong hope :grimacing: ) that the second after we release v0.17.0 we also drop Python 2.6 I worked around that issue by using the original concretizer and avoiding the:
```console
$ spack spec mpileaks
```
we have in the fast shell tests. Honestly I didn't investigate further the SSL error that was reported with Python 2.6, but let me know if I should dig deeper into it.